### ISSUE TITLE
TASK [get tn10rt machines list] failed while enabled composable flag

### ISSUE THREAD

**Issue by pradiptapks** [redhat-performance/jetpack#444]
```
TASK [get tn10rt machines list] **************************************************************************************************************************************************************************************************************
task path: /home/psahoo/jetpack/common.yml:7
Friday 17 June 2022  03:42:16 -0400 (0:00:00.781)       0:05:46.055 *********** 
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "dest": "/home/psahoo/tn10rt.html", 
    "elapsed": 10, 
    "gid": 1002, 
    "group": "psahoo", 
    "invocation": {
        "module_args": {
            "attributes": null, 
            "backup": null, 
            "checksum": "", 
            "client_cert": null, 
            "client_key": null, 
            "content": null, 
            "delimiter": null, 
            "dest": "/home/psahoo/tn10rt.html", 
            "directory_mode": null, 
            "follow": false, 
            "force": false, 
            "force_basic_auth": false, 
            "group": null, 
            "headers": null, 
            "http_agent": "ansible-httpget", 
            "mode": null, 
            "owner": null, 
            "regexp": null, 
            "remote_src": null, 
            "selevel": null, 
            "serole": null, 
            "setype": null, 
            "seuser": null, 
            "sha256sum": "", 
            "src": null, 
            "timeout": 10, 
            "tmp_dest": null, 
            "unsafe_writes": false, 
            "url": "http://wiki.scalelab.redhat.com/1029u/", 
            "url_password": null, 
            "url_username": null, 
            "use_proxy": true, 
            "validate_certs": true
        }
    }, 
    "mode": "0664", 
    "msg": "Connection failure: timed out", 
    "owner": "psahoo", 
    "size": 32898, 
    "state": "file", 
    "uid": 1001, 
    "url": "http://wiki.scalelab.redhat.com/1029u/"
}

```

**Comment by sanjaychari** [comment]
This should be fixed by https://github.com/redhat-performance/jetpack/pull/451.

**Comment by asyedham** [comment]
Closing the issue as it is fixed in #451
